docs/book/basics/status_subresource.md

```diff
@@ -0,0 +1,127 @@
+## Status Subresource 
+By convention, the Kubernetes API makes a distinction between the specification
+of the desired state of an object (a nested object field called "spec") and the 
+status of the object at the current time (a nested object field called "status"). 
+The specification is a complete description of the desired state, including 
+configuration settings provided by the user, default values expanded by the 
+system, and properties initialized or otherwise changed after creation by other 
+ecosystem components (e.g., schedulers, auto-scalers), and is persisted in stable
+storage with the API object. If the specification is deleted, the object will be 
+purged from the system. The status summarizes the current state of the object in 
```

Comment:
> If the specification is deleted, the object will be purged from the system.

This reads really confusing to me.
You mentioned `The specification is a complete description of the desired state ...` above.
I don't quite understand how does `deleting spec` means here.

Reply:
Removed that sentence, wasn't very useful anyways.